Gems/ScriptCanvas/Code/Tests/Framework/ScriptCanvasUnitTestFixture.h

```diff
@@ -15,4 +15,14 @@ namespace ScriptCanvasUnitTest
         : public UnitTest::LeakDetectionFixture
     {
     };
+
+    MATCHER_P(IsClose, expected, "")
+    {
+        #if AZ_TRAIT_USE_PLATFORM_SIMD_NEON
+        constexpr float tolerance = 0.001f;
+        #else
+        constexpr float tolerance = 0.0f;
```

Comment:
So when not using cmd, the result is expected to be "exactly" equal?

That is very strange for a floating point IsClose function.

Reply:
This doesnt work, so I had to change to either a straight _EQ (for non-neon) or an isClose for NEON